v1.1.0/docs/01-why.mdx

```diff
@@ -1,21 +1,26 @@
 # Why?
 
-Edtr.io was born from the needs of the learning platform [serlo.org](https://serlo.org/)
-and with all the experience we gathered from creating educational resources by a diverse group of authors.
+Edtr.io was born from the needs of the learning platform
+[serlo.org](https://serlo.org/) and with all the experience we gathered from
+creating educational resources by a diverse group of authors.
 
-We tried out a lot of other editing libraries before we settled on creating our own.
-Existing rich text libraries (e.g. [Slate](https://www.slatejs.org/), [Draft.js](https://draftjs.org/))
-came closest to what we needed: a way to build intuitive WYSIWYG rich text editors.
+We tried out a lot of other editing libraries before we settled on creating our
+own (like [Slate](https://www.slatejs.org/) or
+[Draft.js](https://draftjs.org/)). However we noticed that the rich text part is
+only a small (albeit very important) foundation of what we need.
 
-After using them for a while, we noticed that the rich text part is only a small (albeit very important) foundation of what we needed, though.
-When creating interactive learning resources, we often need much more than that. Think for example
-about multiple choice exercises and more interactive learning games. Providing an editing experience
-for such use cases often requires us to have a tight control over the user experience. Trying to achieve that with something like Slate was hard, though:
+When creating interactive learning resources, we need more functionality than a
+rich text library can offer. Take for example multiple choice exercises and
```

Comment:
Not a big fan of this wording. It's not like rich-text frameworks can't do that but it would be super annoying. So, for me, it's more that we provide better abstractions suited to those use cases instead of a larger feature set. An analogy would be to compare an Assembly language to a higher-level programming language: You can still do anything in Assembly, but better abstractions will increase your productivity where you don't need the low-level control.

Reply:
`we need a better abstraction than a rich text library can offer`?